dev/sparktestsupport/modules.py

```diff
@@ -715,8 +715,13 @@ def __hash__(self):
         # unittests
         "pyspark.pandas.tests.indexes.test_base",
         "pyspark.pandas.tests.indexes.test_datetime",
-        "pyspark.pandas.tests.test_dataframe",
-        "pyspark.pandas.tests.test_dataframe_slow",
+        # "splits of pyspark.pandas.tests.test_dataframe",
+        "pyspark.pandas.tests.test_dataframe_splits.split_a_b",
+        "pyspark.pandas.tests.test_dataframe_splits.split_c",
+        "pyspark.pandas.tests.test_dataframe_splits.split_d_k",
+        "pyspark.pandas.tests.test_dataframe_splits.split_l_p",
+        "pyspark.pandas.tests.test_dataframe_splits.split_q_r",
+        "pyspark.pandas.tests.test_dataframe_splits.split_s_x",
```

Comment:
Just for extra clarifying, so I means that there is no test names started with `y` and `z`, right?

Reply:
`split_s_x` means `using a prefix from s to x`